include/ignition/common/SystemPaths.hh

```diff
@@ -87,10 +91,12 @@ namespace ignition
       /// \param[in] _filename Name of the file to find.
       /// \param[in] _searchLocalPath True to search in the current working
       /// directory.
+      /// \param[in] _verbose False to omit console messages.
       /// \return Returns full path name to file with platform-specific
       /// directory separators, or empty string on error.
       public: std::string FindFile(const std::string &_filename,
-                                   const bool _searchLocalPath = true) const;
+                                   const bool _searchLocalPath = true,
+                                   const bool _verbose = true) const;
```

Comment:
I think it makes more sense to have _verbose false by default

Reply:
I was keeping the legacy behaviour as default. All other functions are verbose. The reason to turn this off is that it's now being used as a check within another `FindFile` function, so failure isn't exactly an error.

I considered adding the flag to all functions but that seemed overkill and I didn't have an immediate use case for those.